packages/main/test/specs/base/InvisibleMessage.spec.js

```diff
@@ -0,0 +1,34 @@
+const assert = require("chai").assert;
+const PORT = require("../_port.js");
+
+describe("InvisibleMessage", () => {
+	before(() => {
+		browser.url(`http://localhost:${PORT}/test-resources/pages/base/InvisibleMessage.html`);
+	});
+
+    it("Initial rendering", () => {
+        const politeSpan = browser.$(".ui5-invisiblemessage-polite");
+        const assertiveSpan = browser.$(".ui5-invisiblemessage-assertive");
+
+        assert.ok(politeSpan, "polite span is rendered");
+        assert.ok(assertiveSpan, "assertive span not rendered");
+    });
+
+    it("String annoucement", () => {
+        const politeSpan = browser.$(".ui5-invisiblemessage-polite");
+        const assertiveSpan = browser.$(".ui5-invisiblemessage-assertive");
+        const button = browser.$("#announce-button");
+        const checkBox = browser.$("#announce-checkbox");
+
+        browser.execute(() => {
+			document.getElementById("announce-textarea").value = "announcement";
+		});
+
+        button.click();
+        checkBox.click();
+        button.click();
+
+        assert.ok(politeSpan.getHTML().indexOf("announcement") > -1, "value has been announced");
```

Comment:
This is not true `"value has been announced"`
It is true that it has been rendered

Reply:
Done.

Reply:
Please check if you have pushed the changes.